### PR TITLE
added mead animated figure labels

### PR DESCRIPTION
**Pull request by eread-usgs**
@jiwalker-usgs or @jread-usgs sanity check on this file, please. hope I didnt make any mistakes that cause it to break, pretty sure I have all of the figure labels, etc in.

### CODE REVIEW

File: src_data/full_text.en.xml [hunk 1]
```diff
@@ -140,6 +140,105 @@
...
+	
+	<var-data-text-Nevada-normal>plus or minus any ICS created or delivered.</var-data-text-Nevada-normal>
+	
+	<var-data-text-Nevada-shortage-1>(deliveries reduced by 13,000 acre-feet).</var-data-text-Nevada-shortage-1>
+	
+	<var-data-text-Nevada-shortage-2>(deliveries reduced by 17,000 acre-feet).</var-data-text-Nevada-shortage-2>
+	
+	<var-data-text-Nevada-shortage-3>(deliveries reduced by 20,000 acre-feet).</var-data-text-Nevada-shortage-3>
+	
+	<var-data-text-Mexico-flood></var-data-text-Mexico-flood>
```
Comment (jiwalker-usgs) [inline src_data/full_text.en.xml]: this element is empty?

File: src_data/full_text.en.xml [hunk 2]
```diff
@@ -140,6 +140,105 @@
...
+	
+	<var-state-Mexico>Mexico</var-state-Mexico>
+	
+	<var-data-text-Californnia-flood>plus additional deliveries as requested.</var-data-text-Californnia-flood>
+	
+	<var-data-text-Californnia-surplus>plus surplus water.</var-data-text-Californnia-surplus>
+	
+	<var-data-text-Californnia-normal>plus or minus any ICS created or delivered.</var-data-text-Californnia-normal>
+	
+	<var-data-text-Californnia-shortage-1></var-data-text-Californnia-shortage-1>
```
Comment (jiwalker-usgs) [inline src_data/full_text.en.xml]: these are empty?

File: src_data/full_text.en.xml [hunk 3]
```diff
@@ -140,6 +140,105 @@
...
+	
+	<mead-pictogram-legend-units>acre-feet</mead-pictogram-legend-units>
+		
+	<mead-scene-units>acre-feet/year</mead-scene-units>
+			
+	<Mexico-pictos-disclaimer>location of blue boxes does not represent location of water use.</Mexico-pictos-disclaimer>
+				
+	<x-pictogram-label>Lower Colorado River Water Entitlement Holders</x-pictogram-label>
+	
+	<x-pictogram-sub-label>(with an Average Annual Consumptive Use above 15,000 acre-feet)</x-pictogram-sub-label>
```
Comment (jiwalker-usgs) [inline src_data/full_text.en.xml]: I'm not sure we need the parentheses here, but that would be more of a @jread-usgs decision.

Comment (eread-usgs) [inline src_data/full_text.en.xml]: I included all punctuation that was within quotes from Jordan Read's file- up to him if he wants to leave it in or not. He was not consistent with including colons and parentheses.

File: src_data/full_text.en.xml (hunk 2 above)
Comment (eread-usgs) [inline src_data/full_text.en.xml]: Yes, empty for now.

**Comment by eread-usgs** [comment]
@jread-usgs up to you to take a look at this.

File: src_data/full_text.en.xml (hunk 3 above)
Comment (eread-usgs) [inline src_data/full_text.en.xml]: parens are good. leaving those.

**Comment by eread-usgs** [comment]
@jiwalker-usgs @jread-usgs tookout empty keys

**Comment by eread-usgs** [comment]
and @jread-usgs please let me know after you successfully parse it

File: src_data/full_text.en.xml [hunk 4]
```diff
@@ -140,6 +140,101 @@
...
+	
+	<allocation-state-flood>flood</allocation-state-flood>
+	
+	<var-condition-surplus>Quantified and Domestic Surplus</var-condition-surplus>
+	
+	<var-condition-normal>Normal Condition</var-condition-normal>
+	
+	<var-condition-shortage-1>Shortage Condition Level 1</var-condition-shortage-1>
+	
+	<var-condition-shortage-2>Shortage Condition Level 2</var-condition-shortage-3>
```
Comment (jread-usgs) [inline src_data/full_text.en.xml]: tag doesn't close. 

Should be
```
<var-condition-shortage-2>Shortage Condition Level 2</var-condition-shortage-2>
```

**Comment by jread-usgs** [comment]
seems to be the only problem.

**Comment by eread-usgs** [comment]
@jread-usgs thank you. pretty sure its ready now.

**Comment by eread-usgs** [comment]
@jread-usgs before you start working on mead figure you should check out this pr - the spanish xml is now ahead of the english xml.

**Comment by jread-usgs** [comment]
Not seeing a lot of tags that are in the mead animated scene -

e.g., `Flood Control Surplus`. Should these be documented or added to the english xml?

**Pull request merged**